src/databricks/sql/thrift_backend.py

```diff
@@ -733,6 +735,8 @@ def _results_message_to_execute_response(self, resp, operation_state):
             .to_pybytes()
         )
         lz4_compressed = t_result_set_metadata_resp.lz4Compressed
+        # TODO: will this fail if metadata doesn't include `isStagingOperation`?
```

Comment:
is the TODO still relevant?

Reply:
Resolved f6127950d0aaa68575eb24cb0bcceb066b6830e5